### PR TITLE
fix(std): undefined editor during rect polling on mobile

### PR DESCRIPTION
**Pull request by L-Sun**
This PR fixed that try get `EditorHost` before rendering in the rect polling of pointer controller.

```ts
// before eidtor rendering, get `EditorHost` will throw a error
class BlockStdScop {
  constructor(options: BlockStdOptions) {
    this._getHost = () => {
      throw new BlockSuiteError(
        ErrorCode.ValueNotExists,
        'Host is not ready to use, the `render` method should be called first'
      );
    };
  // ...
}

// so this line will throw a error
if (!this._dispatcher.host) return;
```

### CODE REVIEW

**Comment by L-Sun** [comment]
* **#8965** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/8965?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/8965?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `master`
<hr />

## How to use the Graphite Merge Queue

Add the label <kbd>merge</kbd> to this PR to add it to the merge queue.

You must have a Graphite account in order to use the merge queue. Sign up using [this link](https://app.graphite.dev/invite/github/toeverything?ref=merge-queue-instructions-in-stack-comment&prId=8965).

<h2></h2>
<sub>

An organization admin has enabled the [Graphite Merge Queue](https://graphite.dev/docs/graphite-merge-queue) in this repository.

</sub>

<sub>

Please do not merge from GitHub as this will restart CI on PRs being processed by the merge queue.

</sub>

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: packages/framework/block-std/src/event/control/pointer.ts [hunk 1]
```diff
@@ -574,23 +540,55 @@ class PanController extends DualDragControllerBase {
...
   dispose() {
-    this.controllers.forEach(controller => controller.dispose());
+    if (this._pollingInterval !== null) {
+      clearInterval(this._pollingInterval);
+      this._pollingInterval = null;
+    }
   }
 
   listen() {
+    this._startPolling();
```
Comment (L-Sun) [inline packages/framework/block-std/src/event/control/pointer.ts:591]: The `listen()` will invoke after editor has been mounted.

**Review by Saul-Mirone — APPROVED** [review]
LGTM

**Comment by L-Sun** [comment]
### Merge activity

* **Dec 12, 5:53 AM EST**: The merge label 'merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=toeverything&repo=blocksuite) once it meets the requirements.

**Pull request merged**